runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -205,6 +254,11 @@ private void announceTime(String preText, String time)
 			.build());
 	}
 
+	private boolean isInGame()
```

Comment:
What is the point of this method?

Reply:
Adjusted it a bit now and reused it within the plugin. Just checks that the current player is in game. The logic within the plugin is a bit confusing -- going to try and improve it and update the logic more.